### PR TITLE
Ability to pass misc values to summary writer.

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

**Comment by atgeirr** [comment]
Having looked at (only) the single change in EclipseIO.hpp I think that looks fine at least! I'll try to implement TCPU using this as a base.

File: opm/output/eclipse/Summary.cpp [hunk 1]
```diff
@@ -736,13 +733,30 @@ static const std::unordered_map< std::string, ofun > funs = {
...
+  {"TIMESTEP" , UnitSystem::measure::time },
+  {"TCPUTS"   , UnitSystem::measure::time },
+  {"TCPUDAY"  , UnitSystem::measure::time },
+  {"STEPTYPE" , UnitSystem::measure::identity },
+  {"TELAPLIN" , UnitSystem::measure::time }
+};
+  
+  
+
+  inline std::vector< const Well* > find_wells( const Schedule& schedule,UnitSystem::measure:: },
```
Comment (atgeirr) [inline opm/output/eclipse/Summary.cpp]: There is some weirdness here, and it won't compile. Removing the UnitSystem... part makes it compile.

Comment (joakim-hove) [reply opm/output/eclipse/Summary.cpp]: Hmmm - that was an unfortunate fat-fingering; fixed.

**Comment by atgeirr** [comment]
This is finished now in your view, right @joakim-hove?

**Comment by joakim-hove** [comment]
I consider this as finished yes - if you as 'customer' agree I guess we are all set.

**Comment by atgeirr** [comment]
I agree, we have something that can be merged. We could wait until it has been tested though, as discussed in the other PR.

**Pull request merged**